crates/escrow/src/lib.rs

```diff
@@ -578,6 +578,13 @@ impl<T: Config> Pallet<T> {
         let last_point = <PointHistory<T>>::get(<Epoch<T>>::get());
         Self::supply_at(last_point, height)
     }
+
+    /// RPC only
+    pub fn total_locked() -> BalanceOf<T> {
+        Locked::<T>::iter().fold(BalanceOf::<T>::default(), |sum, (_, locked)| {
+            sum.saturating_add(locked.amount)
+        })
+    }
```

Comment:
Not 100% sure about including this, would also ask for @sander2's opinion. There are however two alternatives:
- add a `TotalLocked` storage item
- move the lib call to squid (**recommended**)

Reply:
This is iterating over ~1400 items on Kintsugi so yea I'd agree that it's a good idea to use one of the alternatives suggested by Greg